server/src/core/server/graph/schema/schema.graphql

```diff
@@ -2222,6 +2238,8 @@ type Settings @cacheControl(maxAge: 30) {
   they are enabled and any configured image urls
   """
   flairBadges: FlairBadgeConfiguration
+
+  premoderateEmailAddress: PremoderateEmailAddressConfig
```

Comment:
nit: not a big deal, but could be nice to update `PremoderateEmailAddressConfig` to `PremoderateEmailAddressConfiguration` for consistency with other configs in the file (same note for `PremoderateEmailAddressConfigInput`)

Reply:
updated as well!